### PR TITLE
OCPBUG#16746: Modified steps related to local OCI catalogs.

### PR DESCRIPTION
**Pull request by subhtk**
Version(s): 4.14

Issue: [Ocpbug 16746](https://issues.redhat.com/browse/OCPBUGS-16746)

Link to docs preview: [Command references for oc-mirror](https://62844--docspreview.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html#oc-mirror-command-reference_installing-mirroring-disconnected)
[Including local OCI catalogs](https://62844--docspreview.netlify.app/openshift-enterprise/latest/installing/disconnected_install/installing-mirroring-disconnected.html#oc-mirror-oci-format_installing-mirroring-disconnected)

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/oc-mirror-command-reference.adoc [hunk 1]
```diff
@@ -107,12 +104,13 @@ The following tables describe the `oc mirror` subcommands and flags:
...
-The `--use-oci-feature` flag is deprecated. Use the `--include-local-oci-catalogs` flag instead.
-
 |`-v`, `--verbose` `<int>`
 |Specify the number for the log level verbosity. Valid values are `0` - `9`. The default is `0`.
 
 |===
+
+[NOTE]
+====
+The `--use-oci-feature` flag is deprecated. Use local OCI catalogs.
```
Comment (kasturinarra) [inline modules/oc-mirror-command-reference.adoc:114]: I see `--use-oci-feature` and `--include-oci-catalogs` is deprecated if i am not wrong

File: modules/oc-mirror-oci-format.adoc [hunk 2]
```diff
@@ -73,12 +73,10 @@ mirror:
...
 ----
 $ oc mirror --config=./imageset-config.yaml \ <1>
-  --include-local-oci-catalogs                <2>
-  docker://registry.example:5000              <3>
+  docker://registry.example:5000              <2>
 ----
 <1> Pass in the image set configuration file. This procedure assumes that it is named `imageset-config.yaml`.
-<2> Use the `--include-local-oci-catalogs` flag to enable mirroring local OCI catalogs along with other remote content.
-<3> Specify the registry to mirror the content to. The registry must start with `docker://`. If you specify a top-level namespace for the mirror registry, you must also use this same namespace on subsequent executions.
+<2> Specify the registry to mirror the content to. The registry must start with `docker://`. If you specify a top-level namespace for the mirror registry, you must also use this same namespace on subsequent executions.
```
Comment (kasturinarra) [inline modules/oc-mirror-oci-format.adoc:79]: @subhtk I think registry should be mentioned as below in the imageSetConfig.yaml file and not as  docker:// if i am not wrong. 

the way we include local oci in the imagesetconfig.yaml - with oci://<path-to-local-catalog>

**Review by lmzuccarelli — COMMENTED** [review]
/lgtm

**Pull request merged**